content/en/docs/reference/access-authn-authz/mutating-admission-policy.md

```diff
@@ -0,0 +1,54 @@
+---
+reviewers:
+- deads2k
+- sttts
+- cici37
+title: Mutating Admission Policy
+content_type: concept
+---
+
+<!-- overview -->
+
+{{< feature-state state="alpha" for_k8s_version="v1.32" >}}
+
+This page provides an overview of Mutating Admission Policy.
+
+<!-- body -->
+
+## What is Mutating Admission Policy?
+
+Mutating admission policies offer a declarative, in-process alternative to mutating admission webhooks.
```

Comment:
Rather than compare them to another feature, at beta and beyond these docs should explain what MutatingAdmissionPolicies are and do. The docs we write should make sense even if we were to deprecate or even remove admission webhooks (we won't, but please use that thought experiment as a guide).